libs/stack/stack-ui/src/components/Alerts/components/AlertsItem.tsx

```diff
@@ -22,7 +22,9 @@ const AlertsItem = (props: TAlertsItemProps) => {
               {title}
             </span>
           )}
-          {button && <Button themeName={`${themeName}.button`} tokens={tokens} {...button} />}
```

Comment:
Ça a l'air de fonctionner sans set tabIndex à 0, est-ce que c'est nécessaire pour autre chose? Sinon tout fonctionne bien 👍

Reply:
C'est pour éviter que le Button soit focusable is ce n'est pas la ActiveSlide, mais on pourrait juste ne pas le set le tabIndex j'imagine quand il est actif et ça fonctionnerait aussi

Reply:
Sinon je remarque que en mode pagination, le focus va à l'item du carousel avec les picots de pagination après l'item de carousel, mais c'est l'inverse pour la navigation. Penses-tu que ce serait compliqué a fix? si oui on pourrait juste documenter une autre tâche pour

Reply:
Je suis pas trop sûr comment le fix, parce que on a pas le contrôle sur les boutons de paginations contrairement à ceux de navigation

Reply:
Il faudrait refactor pour utiliser une pagination Custom je pense https://codesandbox.io/p/devbox/swiper-react-pagination-custom-m54gfx?file=/src/App.jsx

Reply:
Même là je sais pas si ça aiderait, faudrait avoir le contrôle du wrapper de la pagination. Je pense qu'il faudrait peut-être refaire une pagination from scratch avec un swiper controller

Reply:
créé une autre tâche pour: https://okamca.atlassian.net/browse/STACK-160